app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    """Base class for vaccine errors"""
+
+
+class NotVaccinatedError(VaccineError):
+    """No vaccine certificate were provided"""
+
+
+class OutdatedVaccineError(VaccineError):
+    """Vaccine certificate is outdated"""
+
+
+class NotWearingMaskError(Exception):
+    """Mask location is unknown, but, for sure, it is not at your face"""
```

Comment:
The docstring for `NotWearingMaskError` is informal and might not clearly convey the error's purpose. Consider revising it to something more straightforward, like 'Visitor is not wearing a mask'.